CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG for ASAB WebUI Components
 
+## 25.5.1
+
+- Remove TreeMenu, Console components. Move TreeMenu to lmio_components_webui (#39)
```

Comment:
Remove TreeMenu and Console component. This has been moved into lmio_webui_components